scripts/cvmfsexec-osg-wrapper

```diff
@@ -87,6 +97,17 @@ set -o nounset
 PS4='+ ${LINENO}: '
 #set -x
 
+[[ $(uname -p) == "x86_64" ]] || fail "Only the x86_64 architecture is supported"
+grep -q '^ID_LIKE=.*rhel.*' /etc/os-release || fail "Only rhel-like distributions are supported"
```

Comment:
Really? This doesn't work for Ubuntu?

Reply:
Yep. `makedist` extracts files from OS-specific RPMs, such as fuse.  For the osg config, it works with EL7 and EL8; for the egi config it also works with SuSE.  Ubuntu support would be a feature request for cvmfsexec.